main.go

```diff
@@ -707,6 +707,117 @@ func postUnlockPageHandler(c *gin.Context, db *sql.DB) {
 	c.Redirect(http.StatusFound, "http://localhost:8000/"+username+"/"+repoName+"/issues/"+IssueNumber)
 }
 
+type Collaborator struct {
+	Username  string
+	UserImage string
+}
+
+func getCollaboratorPageHandler(c *gin.Context, db *sql.DB) {
+
+	_, err := authorize(c, db)
```

Comment:
Collaborator page should be only visible to repo owner.

Reply:
Done